packages/calcite-components/src/components/icon/icon.scss

```diff
@@ -3,36 +3,32 @@
  *
  * These properties can be overridden using the component's tag as selector.
  *
- * @prop --calcite-ui-icon-color: The component's color. Defaults to `currentColor`.
+ * @prop --calcite-ui-icon-color: *deprecated* in favor of `--calcite-icon-color`. The component's color. Defaults to `currentColor`.
+ * @props --calcite-icon-color: defines the color of the component. Defaults to `currentColor`.
+ * @props --calcite-icon-size: defines the size of the component.
```

Comment:
I don't think we should expose size - these are really only supposed to be rendered at the three `scale` values.

Reply:
Just `internal-` is probably fine for this one.